cves/CVE-2012-4386.yml

```diff
@@ -103,9 +107,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    There are unit tests for this module but they did not appear to provide any
```

Comment:
How were the tests changed when this vulnerability was fixed?

Reply:
The system's existing tests were updated, but no new tests were added. The fix changed the way that the system handles token values and token names, so the existing tests were updated to handle that change. Do you still think this counts as an "update to make sure this doesn't happen again?"